### DISCUSSION TITLE
GitHub Actions not running on our organisation repositories

### DISCUSSION THREAD

**Discussion by cfarrend** [community/community#26354] [Actions]
<p>Hi all,</p>
<p>We have noticed since around 9:30 pm UTC today that our GitHub actions in our organisation are not running and are staying in the queued state.</p>
<p>Strangely enough, it is working for my own personal account and haven’t seen too many posts on here regarding such issues. So just confirming whether anyone else has this issue at the moment? We have raised this with GitHub support already just to see what’s happening.</p>
<p>We checked our billing and all seems to be okay, and I’m pretty sure they usually fail and tell you that you are out of credits anyway rather than keep your job in pending.</p>
<p>Our public repoistories have the issue also, so here is an example if anyone wants to see</p>
<p><a href="https://github.com/qctrl/python-open-controls/actions/runs/39026589" rel="nofollow noopener">https://github.com/qctrl/python-open-controls/actions/runs/39026589</a></p>
<p>Thanks</p>

**Comment by cfarrend** [comment]
<p>I also forked our repo just to test and it runs</p>
<p><a href="https://github.com/cfarrend/python-open-controls/actions/runs/39028240" rel="nofollow noopener">https://github.com/cfarrend/python-open-controls/actions/runs/39028240</a></p>

**Comment by BrightRan** [comment]
<p>Recently, some other users also reported the same issue:</p>
<p><a href="https://github.community/t/workflow-does-not-start/17618">Workflow Does Not Start</a></p>
<p><a href="https://github.community/t/github-action-pending-for-a-long-time-for-node-js-org/17625">Github action pending for a long time for Node.js org</a></p>
<p>I have reported this issue to the appropriate engineering team, they will evaluate and investigate the issue. If they have any update, I will notify you in time.</p>

**Comment by cfarrend** [comment]
<p>Just an update,</p>
<p>At about 8:15pm UTC today (23 hours later) all of our queued workflows started running. Seems that this issue has resolved</p>
<p>Very disappointed with no explanations given to us by GitHub yet regarding an almost 24 hour outage of Actions. We are hoping for an explanation in our support ticket</p>

**Comment by BrightRan** [comment]
<p><a class="mention" href="/cfarrend">@cfarrend</a> Sorry for the late reply. And glad to hear that the problem has gone on your side.</p>
<p>Yeah, in the past few days the engineers had taken some measures to let the problem be mitigated.</p>
<p>About the reason of the problem, the problem is still under investigation. If the engineers release any explanation, I will tell you. </p>
<p>Apologize again for the inconvenience has caused to you.</p>

**Comment by cfarrend** [comment]
<p>Thank you <a class="mention" href="/brightran">@brightran</a> </p>
<p>We are definitely awaiting an official response from GitHub regarding this matter, so if you learn anything please let us know.</p>

**Comment by BrightRan** [comment]
<p><a class="mention" href="/cfarrend">@cfarrend</a></p>
<p>I got the following reply from the appropriate engineers:</p>
<p><strong>We had a hiccup on our backend infrastructure that impacted a small percentage of orgs and repos &lt; 0.5% using actions. This resulted in jobs not starting for those repos.</strong></p>

**Comment by BrightRan** [comment]
<p><a class="mention" href="/cfarrend">@cfarrend</a> ,</p>
<p>Just want to say hi and confirm if you have received my previous reply which has provided the explanation for this issue.</p>
<p>Do you have any other question for this issue? Feel free to tell us if you need any help about this ticket.</p>

**Comment by cfarrend** [comment, marked as answer]
<p>Hi <a class="mention" href="/brightran">@brightran</a> </p>
<p>Thanks for your response and help with this issue.</p>
<p>I got an official reply from GitHub support (Feb 28, 11:34PM UTC - 20 days after the incident):</p>
<p>Due to an issue with our backend systems, less than 0.1% of organizations using Actions saw a delay in workflows starting between 02-13-2020 and 02-14-2020. Your organization was identified as one of the potentially impacted ones due to this issue.</p>
<p>The impact was within our SLO (Service Level Objective) and we did not update the public status of Actions. We know how important the reliability of our service is for our customers, and we take this responsibility very seriously and apologize for these disruptions to run workflows.</p>
<p>Our engineering team deployed fixes to mitigate on 02-13-2020, a full fix at 02-14-2020 11:00 UTC after which the backlog of jobs started processing. We have taken steps to ensure we identify these issues and communicate early to the impacted customers.</p>
<p>Our hope is that for these future incidents we can get an update from GitHub, it’s just a very unfortunate situation for us but hopeful in the fact that GitHub can improve from this.</p>
<p>Regards,</p>
<p>Chris</p>

**Comment by benaduggan** [comment]
<p>Hi there, my organization (interviewstreet) is experiencing the same issue today. Running personal actions works, but our org actions are all stalled with the message “Starting your workflow run…”</p>

**Comment by andybirenzi** [comment]
<p>We’re experiencing the same issue with GitHub Actions too</p>

**Comment by cfarrend** [comment]
<p>Same here, it is known by GitHub though I’m pretty sure <a href="https://www.githubstatus.com/incidents/sxthxvdlljk4" rel="noopener nofollow ugc">https://www.githubstatus.com/incidents/sxthxvdlljk4</a></p>
